### ISSUE TITLE
inline styles

### ISSUE THREAD

**Issue by everythinginjs** [mathias-rosa/MyDadJoke#2]
Are you aware of the inline styles in you code ? like [this](https://github.com/mathias-rosa/MyDadJoke/blob/fa56f35d70d5493743f6ddf3059dbdb2f1292431/src/pages/SearchResult.tsx#L7)

**Comment by mathias-rosa** [comment]
Yes, actually because I didn't kwew if it was worth it creating a new styled component for just one div. I think the best options is by using material ui layouts but I'm not sure. Are inlines styles worse that an external css file?

**Comment by everythinginjs** [comment]
Good, the point is being consistent with efficient solution.
it's not about just one div in large scale projects.
but the point is Inline styles increase the HTML file size, which can affect the page load time
you can go with emotion solution in MUI and make a styled component for it EZ.
